supertokens_python/recipe/emailverification/utils.py

```diff
@@ -57,42 +52,52 @@ class ParentRecipeEmailVerificationConfig:
     def __init__(self,
                  get_email_for_user_id: Callable[[str, Dict[str, Any]], Awaitable[str]],
                  override: Union[OverrideConfig, None] = None,
-                 get_email_verification_url: Union[Callable[[
-                     User, Dict[str, Any]], Awaitable[str]], None] = None,
-                 create_and_send_custom_email: Union[Callable[[
-                     User, str, Dict[str, Any]], Awaitable[None]], None] = None
+                 get_email_verification_url: Union[Callable[[User, Dict[str, Any]], Awaitable[str]], None] = None,
+                 create_and_send_custom_email: Union[Callable[[User, str, Dict[str, Any]], Awaitable[None]], None] = None,
+                 email_delivery: Union[EmailDeliveryConfig[TypeEmailVerificationEmailDeliveryInput], None] = None,
                  ):
         self.override = override
         self.get_email_verification_url = get_email_verification_url
-        self.create_and_send_custom_email = create_and_send_custom_email
         self.get_email_for_user_id = get_email_for_user_id
+        self.email_delivery = email_delivery
+        self.create_and_send_custom_email = create_and_send_custom_email
+
+        if create_and_send_custom_email:
+            warn("create_and_send_custom_email is depricated. Please use email delivery config instead")
```

Comment:
We need to mark them as deprecated so that the IDE of the dev complains about it. Not this way